static_substate_db.go

```diff
@@ -9,11 +9,11 @@ import (
 )
 
 var (
-	SubstateDirFlag = cli.StringFlag{
-		Name:  "substatedir",
+	SubstateFlag = cli.StringFlag{
+		Name:  "substate",
```

Comment:
Since we are using `--update-db` and `--deletion-db`. Perhaps, the name here should be `substate-db` ?

Reply:
I thought that too but in our spreadsheet was only `substate` …should I change it?

Reply:
Oh, my bad. Please change it to `--substate-db` for consistency.